recipes/diligent-tools/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+    exports_sources = ["CMakeLists.txt", "patches/**", "BuildUtils.cmake"]
+    short_paths = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package_id(self):
+        if self.settings.compiler == "Visual Studio":
+            if "MD" in self.settings.compiler.runtime:
+                self.info.settings.compiler.runtime = "MD/MDd"
+            else:
+                self.info.settings.compiler.runtime = "MT/MTd"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+        if self.options.shared:
+            raise ConanInvalidConfiguration("Can't build diligent tools as shared lib")
+
+    def requirements(self):
+        self.requires("diligent-core/2.5.1")
+        self.requires("imgui/1.87")
+
+    @property
+    def _diligent_platform(self):
+        if self.settings.os == "Windows":
+            return "PLATFORM_WIN32"
+        elif self.settings.os == "Macos":
+            return "PLATFORM_MACOS"
+        elif self.settings.os == "Linux":
+            return "PLATFORM_LINUX"
+        elif self.settings.os == "Android":
+            return "PLATFORM_ANDROID"
+        elif self.settings.os == "iOS":
+            return "PLATFORM_IOS"
+        elif self.settings.os == "Emscripten":
+            return "PLATFORM_EMSCRIPTEN"
+        elif self.settings.os == "watchOS":
+            return "PLATFORM_TVOS"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        self._cmake.definitions["DILIGENT_INSTALL_TOOLS"] = False
+        self._cmake.definitions["DILIGENT_BUILD_SAMPLES"] = False
+        self._cmake.definitions["DILIGENT_NO_FORMAT_VALIDATION"] = True
+        self._cmake.definitions["DILIGENT_BUILD_TESTS"] = False
+
+        self._cmake.definitions[self._diligent_platform] = True
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("*.hpp", src=self._source_subfolder, dst="include/DiligentTools", keep_path=True)        
+        self.copy(pattern="*.dll", src=self._build_subfolder, dst="lib", keep_path=False)
```

Comment:
```suggestion
        self.copy(pattern="*.dll", src=self._build_subfolder, dst="bin", keep_path=False)
```
DLL files should be copies to bin folder